test/lib/components/Spinner/spinner.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Spinner  from '../../../../src/lib/components/Spinner/Spinner.js'
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import configureStore from 'redux-mock-store';
+
+const initialState = {
+	"map-1": {blockLoad: false, showSpinner: true}
+}
+
+const mockStore = configureStore()
+const store = mockStore(initialState);
+
+describe('Spinner', () => {
+
+    it('renders without crashing', () => {
+        shallow(
+            <Spinner 
+                store={store}
+                mapId='map-1'
+                MAP={{}}
+	        />
+        );
+    });
+
+	it('Spinner component renders correctly', () => {
+        const componentWrapper = mount(
+            <Spinner 
+                store={store}
+                mapId='map-1'
+                MAP={{}}
+            />
+        );
+		expect(toJson(componentWrapper)).toMatchSnapshot();
```

Comment:
@ciremusyoka I'm surprised this tabbing didn't get corrected when linting 🤔

Reply:
Also, could we make sure the tabbing of this document is set to 2 spaces? From this diff it looks like their actual tabs.

Reply:
@cKellyDesign tabbing fixed.